packages/plugins/src/sample-project/sample-foundry-config.ts

```diff
@@ -27,6 +27,9 @@ build_info = true
 extra_output = ['storageLayout']
 fs_permissions=[{access="read", path="./out"}, {access="read-write", path="./cache"}]
 allow_paths = ["../.."]
+// This is a custom option implemented in the Sphinx Foundry fork that causes the create2
+// deployment factory used in broadcasts to also be used in scripts and tests.
+always_use_create_2_factory = true
```

Comment:
> It doesn't look like you can use // for comments in toml files. I copied and pasted this snippet into a foundry.toml then ran forge config --json, which led to an error.
>
> Also, IMO, the phrase "custom option implemented in the Sphinx Foundry fork" implies that this is some sort of temporary solution that will only exist in our fork, when it's actually going to be an option in Foundry soon. Given that the user will likely copy and paste this snippet into their codebase, I think the docs here shouldn't mention Sphinx, and should instead just describe the high-level purpose of the option, since I think that's the intent

Reply:
The `//` comments are probably causing the demo test in CI to fail